jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -57,7 +57,7 @@ def parse_data(app, file_obj, data_type=None, data_label=None):
             spectrum_viewer_reference_name=spectrum_viewer_reference_name,
             uncert_viewer_reference_name=uncert_viewer_reference_name
         )
-    elif isinstance(file_obj, str) and os.path.exists(file_obj):
+    elif isinstance(file_obj, str):
```

Comment:
does this now just allow an error to be raised at the `fits.open` line if a string that is not a valid file path is passed?  Do we need to catch that case and raise a more intuitive error message?

Reply:
No, the "check if file exists" logic never reach this line. It is caught in app level here, so this check is redundant and will never be executed.

https://github.com/spacetelescope/jdaviz/blob/a7837c447c9eab53ae4b65810c1e61115ab81e8c/jdaviz/app.py#L509